src/chi/javascript/components/dropdown.js

```diff
@@ -235,9 +235,6 @@ class Dropdown extends Component {
       Util.addClass(this._elem, CLASS_ACTIVE);
       Util.addClass(this._elem, CLASS_HAS_ACTIVE);
       Util.addClass(this._dropdownElem, CLASS_ACTIVE);
-      if (this._popper) {
```

Comment:
Popper.update must be called to re-position the popover before being shown. If it fails in the console from time to time, please make a better check for the function existance, i.e.:
```Javascript
if (this._popper && typeof this._popper.update === "function") {
  this._popper.update();
}
```

Reply:
I see, thanks!